domain/induction/src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/induction/dto/UpdatePreviousQualificationsDto.kt

```diff
@@ -6,7 +6,7 @@ import java.util.UUID
 
 data class UpdatePreviousQualificationsDto(
   val reference: UUID,
-  val educationLevel: HighestEducationLevel,
+  val educationLevel: HighestEducationLevel?,
```

Comment:
As you'll see below, I've also had to provide a Flyway script to make the corresponding DB column nullable.